test/integration_test.rb

```diff
@@ -5,14 +5,31 @@
 require "fileutils"
 
 module RuboCopRunner
+  MD_LOAD_INLINE_MODE = "inline"
+
   def run_rubocop(path, options: "")
     output, _status = Open3.capture2(
-      "bundle exec rubocop #{options} #{path}",
+      cmd_command_by_env(path, options),
       chdir: File.join(__dir__, "fixtures")
     )
 
     output
   end
+
+  private
+
+  def cmd_command_by_env(path, options)
+    cmd_command = "bundle exec rubocop"
+    load_mode = ENV.fetch("MD_LOAD_MODE", MD_LOAD_INLINE_MODE)
+
+    if load_mode == MD_LOAD_INLINE_MODE
+      md_path = File.expand_path("../lib/rubocop-md.rb", __dir__)
+
+      cmd_command = "#{cmd_command} -r #{md_path}"
+    end
+
+    "#{cmd_command} #{options} #{path}"
+  end
 end
```

Comment:
This is an example of improper usage of the extract method pattern. It breaks cohesion; previously, the code related to the command generation was in a single place; now, we have it spread across the file and we lose the context (what is `-r`, where it comes from)?

Longer methods are sometimes (quite often) better.